<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="MainLayout">
      <item>
       <layout class="QVBoxLayout" name="InputDataLayout">
        <item>
         <layout class="QVBoxLayout" name="HeaderLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="InputDataLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>450</width>
              <height>19</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Input data:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="DataLayout">
          <item>
           <layout class="QVBoxLayout" name="FirstPersonLayout">
            <item>
             <layout class="QHBoxLayout" name="FirstPersonNameLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>First person's name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="firstPersonNameLineEdit">
                <property name="maxLength">
                 <number>50</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LatLongLabels1">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Latitude:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Longitude:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LatLongLayout1">
              <item>
               <widget class="QLineEdit" name="firstPersonLatLineEdit">
                <property name="maxLength">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="firstPersonLongLineEdit">
                <property name="text">
                 <string/>
                </property>
                <property name="maxLength">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="SecondPersonLayout">
            <item>
             <layout class="QHBoxLayout" name="SecondPersonNameLayout">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Second person's name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="secondPersonNameLineEdit">
                <property name="maxLength">
                 <number>50</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LatLongLabels2">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Latitude:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Longitude:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LatLongLayout2">
              <item>
               <widget class="QLineEdit" name="secPersonLatLineEdit">
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="secPersonLongLineEdit">
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="ThirdPersonLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Third person's name:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="thirdPersonNameLineEdit">
                <property name="maxLength">
                 <number>50</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Latitude:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Longitude:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LatLongLayout3">
              <item>
               <widget class="QLineEdit" name="thirdPersonLatLineEdit">
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="thirdPersonLongLineEdit">
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="FourthPersonLayout">
            <item>
             <layout class="QHBoxLayout" name="FourthPersonNameLayout">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Fourth person's location:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="fourthPersonNameLineEdit">
                <property name="maxLength">
                 <number>50</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LatLongLabes4">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Latitude:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Longitude:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LatLongLayout4">
              <item>
               <widget class="QLineEdit" name="fourthPersonLatLineEdit">
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="fourthPersonLongLineEdit">
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="FifthPersonLayout">
            <item>
             <layout class="QHBoxLayout" name="FifthPersonNameLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="text">
                 <string>Fifth person's location:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="fifthPersonNameLineEdit">
                <property name="maxLength">
                 <number>50</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LatLongLabes5">
              <item>
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Latitude:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Longitude:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LatLongLayout5">
              <item>
               <widget class="QLineEdit" name="fifthPersonLatLineEdit">
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="fifthPersonLongLineEdit">
                <property name="maxLength">
                 <number>10</number>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="Line" name="line_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="ResultControlsLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Result window:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="resultList"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
